### PR TITLE
test(JestMigration): convert ZAxisSpec

### PR DESCRIPTION
**Pull request by akib22**
## Description
Refactor `test/specs/cartesian/ZAxisSpec.js` to `test-jest/cartesian/ZAxis.spec.tsx`

## Related Issue
#3195 

## Types of changes
- Jest Migration

## Checklist:
- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.

### CODE REVIEW

File: test-jest/cartesian/ZAxis.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  it("Don't render anything", () => {
+    render(
+      <Surface width={500} height={500}>
+        <ZAxis dataKey="x" name="stature" unit="cm" />
+      </Surface>,
+    );
+    const svg = document.querySelector('svg');
+
+    expect(svg).toBeInTheDocument();
+    expect(svg?.children).toHaveLength(2);
```
Comment (nikolasrieble) [inline test-jest/cartesian/ZAxis.spec.tsx:15]: Why is this condition different to the one in the original test? 

```
    expect(wrapper.find('svg').children.length).to.equal(1);
    expect(wrapper.find('svg noscript').children.length).to.equal(1);
```

**Review by nikolasrieble — CHANGES_REQUESTED** [review]
I am not sure whether the tests are equivalent.

**Comment by akib22** [comment]
image
The above image shows the HTML code of the ZAxis component. There is only one SVG element inside that element that contains two children. So, I can't find any element with this pattern `svg noscript` and can't able to test that condition. So, I have tested it like this. Please let me know what should I do now.

File: test-jest/cartesian/ZAxis.spec.tsx (hunk 1 above)
Comment (ckifer) [reply test-jest/cartesian/ZAxis.spec.tsx:15]: Yeah I tested the output as well - there is no way this test was working properly beforehand. This change works - it just renders an empty `title` and `desc` tag

**Review by ckifer — APPROVED** [review]

**Pull request merged**